include/drivers/watchdog/sunxi-twd.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2017 Samuel Holland <samuel@sholland.org>
+ * Copyright © 2018 Drew Walters <drewwalters96@gmail.com>
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_WATCHDOG_SUNXI_TWD_H
+#define DRIVERS_WATCHDOG_SUNXI_TWD_H
+
+#define R_TWD_CTRL_REG     (0x10)
+#define R_TWD_RESTART_REG  (0x14)
+#define R_TWD_INTV_VAL_REG (0x30)
```

Comment:
These are not part of the API -- they're only used internally by the driver, so they belong in the C file.